### PR TITLE
DRILL-7996: Documentation for OCI OS Storage Plugin

### PR DESCRIPTION
**Pull request by vdiravka**
# [DRILL-7996](https://issues.apache.org/jira/browse/DRILL-7996): Documentation for OCI OS Storage Plugin

## Description

This the DOC page for Drill web-site about Oracle Cloud Infrastructure Object Storage. See [DRILL-7995](https://issues.apache.org/jira/browse/DRILL-7995) for more info.

## Documentation
This changes fully about doc

### CODE REVIEW

File: _docs/en/connect-a-data-source/plugins/111-OCI-OS-storage-plugin.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+To configure the OCI OS storage plugin, log in to the Drill Web UI at `http://<drill-hostname>:8047`. The `drill-hostname` is a node on which Drill is running. Go to the **Storage** page and click **Update** next to the OS storage plugin option or **Create** new if it doesn't exist yet. 
+
+	{
+ 	"type": "file",
+	"connection": "oci://{bucket_name}@{namespace}/",
+	"config": {
+		"fs.oci.client.hostname": "https://objectstorage.us-ashburn-1.oraclecloud.com",
+		"fs.oci.client.auth.tenantId": "ocid1.tenancy.oc1..exampleuniqueID",
+		"fs.oci.client.auth.userId": "ocid1.user.oc1..exampleuniqueID",
+		"fs.oci.client.auth.fingerprint": "20:3b:97:13:55:1c:5b:0d:d3:37:d8:50:4e:c5:3a:34",
```
Comment (cgivre) [inline _docs/en/connect-a-data-source/plugins/111-OCI-OS-storage-plugin.md:29]: @vdiravka 
Is this fingerprint ok to publish here?

File: _docs/en/connect-a-data-source/plugins/111-OCI-OS-storage-plugin.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+	    ...
+	  }  
+
+**Note:** The `"config"` block in the OS storage plugin configuration contains properties to define your OCI credentials. Do not include the `"config"` block in your OS storage plugin configuration if you defined your OCI credentials in the `core-site.xml` file.
+
+To configure the plugin in core-site.xml file, navigate to the $DRILL_HOME/conf or $DRILL_SITE directory, and rename the core-site-example.xml file to core-site.xml
+
+Configure the OS storage plugin configuration to use an external provider for credentials or directly add the credentials in the configuration itself, as described below. Click **Update** to save the configuration when done.
+
+## Providing OCI OS Credentials
```
Comment (cgivre) [inline _docs/en/connect-a-data-source/plugins/111-OCI-OS-storage-plugin.md:42]: I know it is documented elsewhere, but could we provide information about how to use the password vault and/or an example here as well?

**Review by cgivre — CHANGES_REQUESTED** [review]
@vdiravka 
Thanks for writing this up.  A few minor changes, but @dzamo recently changed the way Drill is doing the web site and docs and I believe that this needs to go on this repo (https://github.com/apache/drill-site) rather than the `gh-pages` branch of the Drill repo.

**Comment by dzamo** [comment]
Yes, this PR (and all future website-related PRs) should go to drill-site/master please.

File: _docs/en/connect-a-data-source/plugins/111-OCI-OS-storage-plugin.md (hunk 1 above)
Comment (vdiravka) [reply _docs/en/connect-a-data-source/plugins/111-OCI-OS-storage-plugin.md:29]: Yes, it is :)
It is an example from https://docs.oracle.com/en-us/iaas/Content/API/SDKDocs/hdfsconnector.htm#ariaid-title4

File: _docs/en/connect-a-data-source/plugins/111-OCI-OS-storage-plugin.md (hunk 2 above)
Comment (vdiravka) [reply _docs/en/connect-a-data-source/plugins/111-OCI-OS-storage-plugin.md:42]: I'll set up Vault and add the plugin configs example here then

**Comment by vdiravka** [comment]
> @vdiravka
> Thanks for writing this up. A few minor changes, but @dzamo recently changed the way Drill is doing the web site and docs and I believe that this needs to go on this repo (https://github.com/apache/drill-site) rather than the `gh-pages` branch of the Drill repo.

Ah, exactly. Forgot about that. I'll add info about Vault Credential Provider and reopen it for https://github.com/apache/drill-site

**Pull request closed without merging**